packages/graphiql/src/index.tsx

```diff
@@ -122,8 +122,9 @@ const subscribeWithMultipart = (
 
     try {
       if (isAsyncIterable(maybeStream)) {
-        for await (const { body: chunk } of maybeStream) {
-          if (typeof chunk !== "string") {
+        for await (const part of maybeStream) {
+          if (part.json) {
+            const chunk = part.body;
```

Comment:
we only need to do this, coz TypeScript cant discriminate the union if we destruct.